subprojects/test-kit/src/main/java/org/gradle/testkit/runner/internal/ToolingApiGradleExecutor.java

```diff
@@ -125,7 +124,7 @@ public GradleExecutionResult run(GradleExecutionParameters parameters) {
 
             launcher.run();
         } catch (UnsupportedVersionException e) {
-            throw new InvalidRunnerConfigurationException("The build could not be executed due to a feature not being supported by the target Gradle version", e);
+            throw e;
```

Comment:
❌ This should not change, it is unrelated to the deprecation.

Reply:
Thanks. Reverted.